### ISSUE TITLE
Equation for loss

### ISSUE THREAD

**Issue by oakif** [whitead/dmol-book#3]
In section B 1.3.4, the loss function is defined as `jnp.sqrt(jnp.mean((y - labels)**2))` where y is the predictions from the linear model. Not that here, the loss is equal to the square root of the mean. However, in equation 1.16, the loss seems to be defined as simply the mean (of the square of the differences)—no square root involved. I presume that equation 1.16 needs to have L² on the LHS for the loss definitions to be consistent.

**Comment by whitead** [comment]
Thanks, fixed in bd8707e41f618bacc7695e39d5268a46ed21f410
